spaces/zzz-finance/index.json

```
@@ -27,6 +27,20 @@
     "defaultTab": "all",
     "minScore": 20,
     "onlyMembers": false
```

You will miss a comma here.
```suggestion
    "onlyMembers": false,
```

---

> You will miss a comma here.

Edited to include comma

---

Also, I think your Gitbook needs to be updated, in the example used demonstrating how to add a space, there is no comma after the onlyMembers parameter input. https://docs.snapshot.page/guides/create-a-space